native-tls-hook.js

```diff
@@ -56,6 +61,23 @@ TARGET_LIBS.forEach((targetLib) => {
     }
 });
 
+function hook_proxygen_SSLVerification(library) {
+    const functionName = "_ZN8proxygen15SSLVerification17verifyWithMetricsEbP17x509_store_ctx_stRKNSt6__ndk112basic_stringIcNS3_11char_traitsIcEENS3_9allocatorIcEEEEPNS0_31SSLFailureVerificationCallbacksEPNS0_31SSLSuccessVerificationCallbacksERKNS_15TimeUtilGenericINS3_6chrono12steady_clockEEERNS_10TraceEventE";
```

Comment:
Any idea how consistent this name is? It looks like the kind of thing that might change frequently between releases.

Can we match against this instead, with something like `*proxygen*SSLVerification*verifyWithMetrics*FailureVerificationCallbacks*`? That seems likely to be sufficiently specific but much less likely to change in future.

Reply:
I don't know, this method name is taken from libliger.so. It works.